### ISSUE TITLE
Character Directory breaks if you sort by ERP tag

### ISSUE THREAD

**Issue by Burrito-Justice** [VOREStation/VOREStation#9697]
#### Brief description of the bug
The new TGUI character directory is pretty great and clean. However, it appears that clicking on the ERP tag filter cuts straight to a bluescreen (all other filters appear fine). Note that this was tested from the lobby screen, and not while in game, though I doubt this has any real bearing on the issue.

#### Steps to reproduce
- Open Character-Directory
- Sort by ERP
- AIIIIIEEEEEEEEEEEEEEEEEE

#### Code Revision
5e4bf92bbb122677f2843ae82ba2adcf856f1943
 

#### Anything else you may wish to add:
```
TypeError: Unable to get property 'localeCompare' of undefined or null reference
   at Anonymous function (http://127.0.0.1:6765/tmp2564/tgui.bundle.js:35:143601)
   at d (http://127.0.0.1:6765/tmp2564/tgui.bundle.js:35:142982)
   at Anonymous function (http://127.0.0.1:6765/tmp2564/tgui.bundle.js:31:73461)
   at Pe (http://127.0.0.1:6765/tmp2564/tgui.bundle.js:31:73224)
   at Anonymous function (http://127.0.0.1:6765/tmp2564/tgui.bundle.js:31:76856)
   at Oe (http://127.0.0.1:6765/tmp2564/tgui.bundle.js:31:76751)
   at Anonymous function (http://127.0.0.1:6765/tmp2564/tgui.bundle.js:31:73983)
   at Pe (http://127.0.0.1:6765/tmp2564/tgui.bundle.js:31:73739)
   at Oe (http://127.0.0.1:6765/tmp2564/tgui.bundle.js:31:74264)
   at Anonymous function (http://127.0.0.1:6765/tmp2564/tgui.bundle.js:31:72724)
User Agent: Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0E; .NET4.0C; .NET CLR 3.5.30729; .NET CLR 2.0.50727; .NET CLR 3.0.30729; Tablet PC 2.0; Zoom 3.6.0)
```

**Comment by Burrito-Justice** [comment]
Apparently TGUI will bluescreen if you sort by vore, too, but only sometimes. This happened while in a NIF, but again, uncertain if that has any bearing.
```TypeError: Unable to get property 'localeCompare' of undefined or null reference
   at Anonymous function (http://127.0.0.1:44055/tmp2272/tgui.bundle.js:35:143601)
   at d (http://127.0.0.1:44055/tmp2272/tgui.bundle.js:35:142982)
   at Anonymous function (http://127.0.0.1:44055/tmp2272/tgui.bundle.js:31:73461)
   at Pe (http://127.0.0.1:44055/tmp2272/tgui.bundle.js:31:73224)
   at Anonymous function (http://127.0.0.1:44055/tmp2272/tgui.bundle.js:31:76856)
   at Oe (http://127.0.0.1:44055/tmp2272/tgui.bundle.js:31:76751)
   at Anonymous function (http://127.0.0.1:44055/tmp2272/tgui.bundle.js:31:73983)
   at Pe (http://127.0.0.1:44055/tmp2272/tgui.bundle.js:31:73739)
   at Oe (http://127.0.0.1:44055/tmp2272/tgui.bundle.js:31:74264)
   at Anonymous function (http://127.0.0.1:44055/tmp2272/tgui.bundle.js:31:72724)
User Agent: Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0E; .NET4.0C; .NET CLR 3.5.30729; .NET CLR 2.0.50727; .NET CLR 3.0.30729; Tablet PC 2.0; Zoom 3.6.0)```

also the default ERP setting is "non-erp" instead of "unset" which is probably unintentional

**Comment by H0lySquirr3l** [comment]
![image](https://user-images.githubusercontent.com/34927367/108759428-f168c080-7511-11eb-8501-e4d588cd8361.png)

![image](https://user-images.githubusercontent.com/34927367/108759451-f9c0fb80-7511-11eb-9f06-9236c2cbec64.png)


Checked both in menu and as ghost. No error was given so this should be closed

**Comment by KillianKirilenko** [comment]
> Checked both in menu and as ghost. No error was given so this should be closed

Don't jump to conclusions, please. No PRs have been made explicitly fixing this.

Now, if I had to guess, this is happening because the default state for unset tags is null if you're loading a character created before the directory overhaul or the addition of the ERP Tag column. If that's correct, then the reason you're seeing this less is because more people are updating their prefs/character saves.

Basically, it should naturally resolve over time; as soon as you hit 'Save Slot' you get the proper default values for the tags set.

Keep your eyes peeled for anyone with a blank Vore Tag and try sorting by that column, see what happens. I bet it'll bluescreen just as readily as it would for the ERP Tag column.

**Comment by KotetsuRedwood** [comment]
I can confirm I've seen bluescreens when trying to sort when someone has a null tag even before ERP column - it's definitely a null-variable problem. 

Stacktrace having `TypeError: Unable to get property 'localeCompare' of undefined or null reference` definitely confirms that theory, too!
